### PR TITLE
upgrading pyarrow to fix the numpy 1.21.0 broken changes and fixing integ tests

### PR DESCRIPTION
**Pull request by NikhilRaverkar**
*Issue #, if available:*
1. Updated pyarrow version to 3.0.0 to fix the broken changes with numoy 1.21.0
2. Fixed broken integ tests

*Description of changes:*
[Fixed pyarrow version ](https://github.com/Azure/MachineLearningNotebooks/issues/1314)

*Testing:*
Verified that error does not exist anymore after the upgrade:
```
>>> import pandas as pd
>>> df = pd.DataFrame({'x': [1,2]})
>>> df.to_parquet('test.parquet', engine='pyarrow')
>>> print(pd.__version__)
1.1.3
>>> print(pyarrow.__version__)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'pyarrow' is not defined
>>> import pyarrow
>>> print(pyarrow.__version__)
4.0.0
>>> import numpy
>>> print(numpy.__version__)
1.21.0
>>> 
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/integration/test_multiple_model_endpoint.py [hunk 1]
```diff
@@ -55,19 +55,24 @@ def modulevolume():
...
+
     test_name = 'sagemaker-sklearn-serving-test'
     try:
         command = (
             'docker run --name {} -p 8080:8080'
-            ' --mount type=volume,source=dynamic_endpoint_model_volume,target=/opt/ml/model,readonly'
-            ' --mount type=volume,source=dynamic_endpoint_module_volume,target=/user_module,readonly'
+            # ' --mount type=volume,source=dynamic_endpoint_model_volume,target=/opt/ml/model,readonly'
+            # ' --mount type=volume,source=dynamic_endpoint_module_volume,target=/user_module,readonly'
+            ' -v {}:/opt/ml/model'
```
Comment (secastillo) [inline test/integration/test_multiple_model_endpoint.py:67]: for my understanding, why were these arguments `module_dir` and `model_dir` added with these changes?

File: test/integration/test_multiple_model_endpoint.py [hunk 2]
```diff
@@ -55,19 +55,24 @@ def modulevolume():
 
 @pytest.fixture(scope='module', autouse=True)
```
Comment (dewan-c) [inline test/integration/test_multiple_model_endpoint.py:56]: Do we also want to update the version in test setup(tox.ini)?

Comment (NikhilRaverkar) [reply test/integration/test_multiple_model_endpoint.py:56]: Good catch! updated

**Comment by NikhilRaverkar** [comment]
Closing this since PR#110 is upgrading the version. Will create a new PR for the new version

**Pull request closed without merging**